plugins/org.jkiss.dbeaver.ext.generic/plugin.xml

```diff
@@ -1507,6 +1488,15 @@
                         categories="nosql,timeseries,aws">
                     <notAvailableDriver title="%not.available.title" description="%not.available.description" message="%driver.timestream.reason"/>
                 </driver>
+                <driver
+                        id="neo4j-ce"
```

Comment:
WE shouldn't change old driver ID.
Users may have this config already, it must keep working when they switch to PRO.